entry_types/scrolled/package/src/frontend/Image.js

```diff
@@ -19,17 +21,26 @@ export function Image(props) {
     const focusY = typeof image.configuration.focusY === 'undefined' ? 50 : image.configuration.focusY;
 
     return (
-      <img className={classNames(styles.root)}
-           src={image.urls[props.variant]}
-           style={{
-             objectPosition: `${focusX}% ${focusY}%`
-           }} />
+      <>
+        <img className={classNames(styles.root)}
+             src={image.urls[props.variant]}
+             style={{
+               objectPosition: `${focusX}% ${focusY}%`
+             }} />
+        {renderStructuredData(props, image)}
```

Comment:
This maybe is a matter of taste, but I don't think `{props.structuredData && image ? <ImageStructuredData file={file} /> : null}` or  `{props.structuredData && image && <ImageStructuredData file={file} />}` are real improvements here. Instead I also extracted rendering the `img` element into a separate function to obtain a consistent level of abstraction.

Reply:
I'd say the second version you provide in your comment is a real improvement, because it is a one-liner that makes clear in which case structured data is added.

I agree that `renderImageTag` is a good simplification of the render method. 

Considering what happens in `renderStructuredData`, and especially considering it receives all the props but only uses one prop, to me, it looks like obfuscation of what's happening. At the same time, having consistent conventions for Pageflow's React code would indeed be a nice thing.

To get around the double `&&`, you could introduce `const renderStructuredData = props.structuredData && image`

Reply:
I see your point. Still, one could argue the other way around: `Image` renders two distinct things: the image tag and the structured data. The main render function makes that clear now. Both parts rely at least on some props and hence all of the props are available in both parts. Structured data currently just uses one, but there could be more down the road. The file lookup is pulled up into the main render method since it obtains a shared piece of information.

Reply:
Well, I definitely see that one of the more annoying activities when using React can be to reshuffle interfaces as components and functions grow to be more complex. Indeed, `renderStructuredData` optimizes for extensibility in that way.

If I frame this as a decision between momentary readability and future convenience in extensibility, I still think this is a premature consideration, and I remain convinced that we are hiding what component we actually use here behind an excessive level of abstraction. Still, in case I'm the one who extends this functionality one day, I might be the one cursing because I'd have to add this level of abstraction back in.

I won't press the issue any further